ipn/backend.go

```diff
@@ -92,6 +93,48 @@ type Notify struct {
 	// type is mirrored in xcode/Shared/IPN.swift
 }
 
+func (n Notify) String() string {
+	s := "Notify{"
```

Comment:
strings.Builder to avoid the quadratic garbage of += on strings. You can Fprintf to it.